### PR TITLE
Minor edits to README.md (mostly formatting)

### PR DESCRIPTION
**Pull request by Zearin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -281,17 +290,17 @@ A. Probably because the database is being queried every time and this uses a
...
    in which case it will still use the database to ensure accuracy.
 
-Q. bash-completion interferes with my `command_not_found_handler` function!
+**Q. bash-completion interferes with my `command_not_found_handler` function!**
 
 A. If your `command_not_found_handler` function is not intended to
    address (possibly missing) commands invoked during bash
-   programmable completion functions, you can take that into account
-   in your function for example by testing if the `$COMP_`* variables
+   programmable completion functions. You can account for this
```
Comment (scop) [inline README.md]: Replacing the comma with the period and splitting the sentence like that (before "You") seems clunky here. Was this intentional? Or did you also intend to remove the "If" from the start of the sentence?

File: README.md [hunk 2]
```diff
@@ -318,9 +327,9 @@ A. The readline(3) library offers a few settings that can make tab
...
    completion lists.
 
-Q. Is bash the be-all-and-end-all of completion as far as shells go?
+**Q. Is bash the be-all-and-end-all of completion as far as shells go?**
 
 A. Absolutely not. zsh has an extremely sophisticated completion system
    that offers many features absent from the bash implementation. Its
    users often cannot resist pointing this out. More information can
-   be found at: http://www.zsh.org/
+   be found at: http://www.zsh.org
```
Comment (scop) [inline README.md]: I'd rather leave this as it was, with the trailing slash.

File: README.md (hunk 1 above)
Comment (Zearin) [inline README.md]: Whoops—I agree.  I think I was looking at the effect on the second sentence, but forgot to check the first.  I’ll fix shortly!

File: README.md (hunk 2 above)
Comment (Zearin) [inline README.md]: K.

**Comment by scop** [comment]
Thanks.

**Pull request merged**